doc/dev_guide/lazy_computations.rst

```diff
@@ -46,6 +46,20 @@ cases for lazy signals will have to be written. If a function operates on
 arbitrary-sized arrays and the shape of the output can be known before calling,
 ``da.map_blocks`` and ``da.map_overlap`` are efficient and flexible.
 
+For more recent versions of dask (dask>2021.11) when using hyperspy in a jupyter
+notebook a helpful html representation is available.
+    .. code-block::python
+        import numpy as np
+        import hyperspy.api as hs
+        x = np.zeros((20,20,10,10,10,))
+        x = hs.signals.Signal2D(x)
+        display(x)
+
+.. figure:: images/chunks.png
+
+This helps to visualize the chunk structure and identify axes where the chunk spans the entire
+axis (bolded axes).
+
```

Comment:
Not sure.  On one hand I think that using the html representation is very helpful when using lazy signals and I'm not quite sure how familiar our user base is with lazy computations.  That being said this should be very intuitive so this might be a little much.  Little things like bolding axes where the chunk spans the entire axis might be less intuitive.